internal/pkg/template/templates/workloads/partials/cf/autoscaling.yml

```diff
@@ -296,4 +296,56 @@ AutoScalingPolicy{{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}Ev
 {{- end }}{{/* range $topic := .Subscribe.Topics */}}
 {{- end }}{{/* if .Subscribe */}}
 
-{{- end }}{{/* if .Autoscaling.QueueDelay */}}
\ No newline at end of file
+{{- end }}{{/* if .Autoscaling.QueueDelay */}}
+
+{{- if .Autoscaling.Requests}}
+AutoScalingPolicyALBSumRequestCountPerTarget:
+  Type: AWS::ApplicationAutoScaling::ScalingPolicy
+  Properties:
+    PolicyName: !Join ['-', [!Ref WorkloadName, ALBSumRequestCountPerTarget, ScalingPolicy]]
+    PolicyType: TargetTrackingScaling
+    ScalingTargetId: !Ref AutoScalingTarget
+    TargetTrackingScalingPolicyConfiguration:
+      CustomizedMetricSpecification:
+        Dimensions:
+          - Name: LoadBalancer
+            {{- if eq .WorkloadType "Backend Service"}}
+            Value: !GetAtt EnvControllerAction.InternalLoadBalancerFullName
+            {{- else}}
```

Comment:
Just curious why we didn't have `requests`/`response_time` for Worker Services previously...?

Reply:
They are not behind load balancers - requests and response_time are metrics that are emitted by ALB - worker services receive requests from queues ✌️

Reply:
So then this should be an `{{- if...}}` not an `{{- else}}` because the worker service cfn template includes this partial.

Reply:
Those cases will never occur - `Validate()` will error out if `requests` or `response_time` are set on a Worker Service (see [here](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/manifest/validate.go#L832-L879))

Reply:
Gotcha, gotcha. Thanks!